nix/vast.nix

```diff
@@ -46,30 +48,27 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ cmake ];
   propagatedNativeBuildInputs = [ pkgconfig pandoc ];
-  buildInputs = [ libpcap ];
+  buildInputs = [ libpcap jemalloc ];
   propagatedBuildInputs = [ arrow-cpp caf ];
 
   cmakeFlags = [
     "-DCAF_ROOT_DIR=${caf}"
     "-DVAST_RELOCATABLE_INSTALL=OFF"
     "-DVAST_VERSION_TAG=${version}"
+    "-DVAST_USE_JEMALLOC=ON"
     # gen-table-slices runs at build time
     "-DCMAKE_SKIP_BUILD_RPATH=OFF"
   ] ++ lib.optionals static [
     "-DVAST_STATIC_EXECUTABLE:BOOL=ON"
     "-DZSTD_ROOT=${zstd}"
-  ];
+  ] ++ lib.optional disableTests "-DBUILD_UNIT_TESTS=OFF";
```

Comment:
Side note: not sure if `BUILD_UNIT_TESTS` actually works, it was broken a while ago.

Reply:
Thanks for the heads-up!